src/app/migration/src/src/app/modules/workspace/components/create-editors/collection-editor/collection-editor.component.ts

```diff
@@ -1,15 +1,632 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, AfterViewInit, NgZone } from '@angular/core';
```

Comment:
why do we need NgZone

Reply:
We are using onClosing function to close the modal, the angular function written for this jquery plugin event is not executed. so using ngZone we can execute our angular code inside the jquery event.